extensions/stackdriver/stackdriver.h

```diff
@@ -80,7 +80,11 @@ class StackdriverRootContext : public RootContext {
   void deleteFromTCPRequestQueue(uint32_t id);
   void incrementReceivedBytes(uint32_t id, size_t size);
   void incrementSentBytes(uint32_t id, size_t size);
+  void incrementTotalReceivedBytes(uint32_t id, size_t size);
```

Comment:
Why not reuse the existing functions?

Reply:
Because the metrics are different... this is incrementing tcp_total_received_bytes...

Reply:
You can increment total_x_bytes inside the existing function.

Reply:
This is not fixed yet.

Reply:
yup. thanks.. fixed

Reply:
I don't see a new commit, did you push?

Reply:
weird.. change was there in the branch bot not in PR.. I pushed again with a comment.. PTAL